os/net/netmgr/netdev_lwip.c

```diff
@@ -70,9 +75,58 @@ static inline int _netif_down(struct netif *ni)
 	return 0;
 }
 
-#define LW_GETND(nic) (struct netdev *)(*(struct netdev **)(&((char *)nic)[sizeof(struct netif)]))
 
+#ifdef CONFIG_NET_NETMGR_ZEROCOPY
+static err_t _lwip_linkoutput(struct netif *nic, struct pbuf *buf)
+{
+	struct netdev *dev = LW_GETND(nic);
 
+	int res = ND_NETOPS(dev, linkoutput)(dev, (void *)buf, 0);
+	if (res < 0) {
+		return ERR_IF;
+	}
+
+	return ERR_OK;
+}
+
+
+static int lwip_input(struct netdev *dev, void *frame_ptr, uint16_t len)
+{
+	(void)len;
```

Comment:
I think this is not needed.

Reply:
I did it to show that it is unused variable when ZEROCOPY is enabled